hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1439,6 +1439,17 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.lease.hard.limit</name>
+    <value>7d</value>
+    <tag>OZONE, OM, PERFORMANCE</tag>
+    <description>
+      Controls how long an open hsync key is considered as active. Specifically, if a hsync key
+      has been open longer than the value of this config entry, that open hsync key is considered as
+      expired (e.g. due to client crash). Unit could be defined with postfix (ns,ms,s,m,h,d)
```

Comment:
What happens if an admin misconfigures this to `7ms`? A key will almost immediately get committed after a client calls `hsync()`?

Reply:
The point is, should we have a lower limit on this config?

Reply:
We can add one check, the hard limit should not be less than the soft limit.

Reply:
@ChenSammi I agree that it is generally hard to prevent all misconfigurations.

I was prompted when I saw these `ns,ms` units in the description. :D

Reply:
Added check for hard limit should not be less than the soft limit, and made it equal if it is so.